src/About/index.jsx

```diff
@@ -0,0 +1,42 @@
+import React from 'react';
+import './style.scss';
+import divider from '../image/Divider.png';
+import cookImg from '../image/cook.png';
+import dish from '../image/dish.png';
```

Comment:
Usually, fonts, imgs, etc are added to assets folder

Reply:
done